### ISSUE TITLE
Pagination should not show if the page count is zero

### ISSUE THREAD

**Issue by asifrahman01** [AdeleD/react-paginate#160]
Currently the pagination shows(with text previous and next) even if the page count is zero. Ideally it should get hidden if the page count is zero.

**Comment by benj3** [comment]
Im using something similar to this:

`containerClassName={(this.state.arr.length == 0 ? "pagination hidden" : "pagination justify-content-center" )}`

Works fine, In a perfect world it wouldn't render if there was no results but this is a little work around.

**Comment by agierlicki** [comment]
I wrapped the paginate element in a small functional component that does some calculations for the pages and only displays the Pagination if there is data:

```javascript
const Pagination = (props) => {
  if (props.data.length === 0) {
    return null;
  }
  return <Paginate />;
}
```

**Comment by lewisdonovan** [comment]
Found a nice way of doing this with styled-components, you just pass the page count as a prop to a wrapping div, then hide if it's falsy (0).

```
import React from 'react';
import styled, { css } from 'styled-components';
import ReactPaginate from 'react-paginate';

class MyPage extends React.Component {
  render() {

    const paginateWrap = styled.div`
      ${props => props.pageCount ? 'display: block' : display: none'};
    `;

    <PaginateWrap pageCount={this.state.pageCount}>
      <ReactPaginate {...props} />
    </PaginateWrap>

  }
}

export default MyPage;
```

**Comment by MonsieurV** [comment]
I agree, we could have an option to enable that behavior. (and without requiring wrappers)

**Comment by mkelley33** [comment]
> Currently the pagination shows(with text previous and next) even if the page count is zero. Ideally it should get hidden if the page count is zero.

Even if the page count isn't zero, it would be nice if there was a way to hide the numbers altogether, and just show next and previous buttons.
